transactions/btc.ts

```diff
@@ -65,6 +65,15 @@ export function selectUnspentOutputs(
     sumValue += pinnedOutput.value;
   }
 
+  // Sort UTXOs based on block time in ascending order
+  unspentOutputs.sort((a, b) => {
+    if (a.status.block_time && b.status.block_time) {
+      return a.status.block_time - b.status.block_time;
+    }
+    // Handle cases where block time is missing
+    return 0;
```

Comment:
I don't think we should return a 0 here. It could result in things going out of order again and causing similar issues and it would also result in things without a block time randomly being placed in the ordered UTXOs that have a block time.

Instead, we should do something like:
```javascript
if (a.status.block_time) {
  return 1;
} 
if (a.status.block_time) {
  return -1;
} 

return a.amount - b.amount; // If neither have block time, then place them at the end of the list ordered by amount so we use UTXOs with higher amount first
```